brainstormforce/astra
# fix: Removed the Button Border Popup & added color group to the Global Button Border Colors

### Description
 - Trello task - https://trello.com/c/ViOQ1rNc/1111-popout-all-border-settings-in-buttons

### Types of changes
 Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
 - Check the Button Border settings working or not.
 - Check new Boutton border colors option.
 - Check backward compatibility.

### Checklist:
